exasol_integration_test_docker_environment/lib/base/db_os_executor.py

```diff
@@ -87,10 +93,28 @@ def __del__(self):
         self.close()
 
     def exec(self, cmd: str) -> ExecResult:
-        result = self._connection.run(cmd)
+        result = self._connection.run(cmd, warn=True, hide=True)
         output = result.stdout.encode("utf-8")
```

Comment:
It is probably a good idea that you assume combine_stderr to be True here, otherwise stderr wouldn't be merged into stdout. https://docs.fabfile.org/en/1.12.1/usage/interactivity.html#crossing-the-streams

Reply:
? OK .... I did not think too much about the consequences and alternatives here. 
I just identified that we neither need nor like to have a stdout here but we really need to get a result with exit code and stdout (maybe also with stderr) to mimic the behavior of `DockerContainer.exec_run()`.